src/connectors/hits-per-page/connectHitsPerPage.js

```diff
@@ -108,8 +108,13 @@ export default function connectHitsPerPage(renderFn, unmountFn = noop) {
     }
 
     const defaultItems = items.filter(item => item.default === true);
+    const defaultItemsCount = defaultItems.length;
 
-    if (defaultItems.length > 1) {
+    if (items.length && defaultItemsCount === 0) {
```

Comment:
why is there a  check for items.length here?

Reply:
Otherwise an empty array would error:

```js
hitsPerPage({
  items: [],
});
```

Reply:
is this a valid use case though?

Reply:
It works in React InstantSearch. I'm not sure it makes too much sense though.

Reply:
Personally I'm okay with throwing an error, since I don't see what use it can have. Even if you'd set the defaultRefinement, you still need items to apply it

Reply:
Addressed in 3f744e2d4dfabd1b4c3ce61cc76c8fadae53ec7e.